adal/build.gradle

```diff
@@ -131,7 +131,7 @@ android {
     useLibrary 'android.test.mock'
 }
 
```

Comment:
can you add same comment here (dev vs release branch), as in the other libs..

Reply:
Done